thirdeye-server/src/main/java/ai/startree/thirdeye/resources/AlertResource.java

```diff
@@ -148,11 +151,15 @@ protected AlertApi toApi(final AlertDTO dto) {
   @Timed
   @Produces(MediaType.APPLICATION_JSON)
   @Deprecated(forRemoval = true)
-  public Response getInsights(@ApiParam(hidden = true) @Auth ThirdEyePrincipal principal,
-      @PathParam("id") final Long id) {
+  public Response getInsights(
+      @ApiParam(hidden = true) @Auth ThirdEyePrincipal principal,
+      @PathParam("id") final Long id,
+      @Context java.net.http.HttpHeaders httpHeaders
```

Comment:
you can probably simplify the qualified import

Reply:
This file is also importing `javax.ws.rs.core.HttpHeaders`, altho I'm not really sure if there's a difference.